modules/Microsoft.CDN/profiles/endpoint/deploy.bicep

```diff
@@ -0,0 +1,67 @@
+@description('Required. Name of the CDN profile which is unique within the resource group.')
+param profileName string
+
+@description('Required. Name of the endpoint under the profile which is unique globally.')
+param endpointName string
+
+@description('Optional. Resource location.')
```

Comment:
Do you need the location in the child?

Reply:
The value required by the child module must be calculated at the start of the deployment. Params would make sense for this no?

Reply:
Right. Just checked the template doc reference and yes its there. So all good. Just one thing I noticed and not sure if I mentioned it before, is that the folder name for the child resource is `endpoint` not `endpoints`. And as per the doco it should be `endpoints`

![image](https://user-images.githubusercontent.com/28486158/210491510-97087397-cf46-4199-a672-c596949c9080.png)

Please change that to align with the template doco.

Reply:
Changes to `endpoints` in latest commit